phase/generator.go

```diff
@@ -154,10 +162,23 @@ func (g *Generator) getGenerateInputs() buildpack.GenerateInputs {
 func (g *Generator) copyDockerfiles(dockerfiles []buildpack.DockerfileInfo) error {
 	for _, dockerfile := range dockerfiles {
 		targetDir := filepath.Join(g.GeneratedDir, dockerfile.Kind, launch.EscapeID(dockerfile.ExtensionID))
-		var targetPath = filepath.Join(targetDir, "Dockerfile")
-		if dockerfile.Kind == buildpack.DockerfileKindRun && dockerfile.Ignore {
+		targetPath := filepath.Join(targetDir, "Dockerfile")
```

Comment:
Just a nit, but could we move the "target" code below the Platform API 0.13 block? This stuff:

```
		targetDir := filepath.Join(g.GeneratedDir, dockerfile.Kind, launch.EscapeID(dockerfile.ExtensionID))
		targetPath := filepath.Join(targetDir, "Dockerfile")
		if ignoreDockerfile {
			targetPath += ".ignore"
		}
```

I think that will make it easier to understand what's going on when viewing with fresh eyes